Update simplified Chinese localization

Creeper-banner opened the pull request:
Changes: 
Added new localizations for strings that haven't been translated
Fixed some mistakes (e.g. typo, mistranslation and some apparent mistakes due to carelessness)
Better "lore book" translation; now the contents of the book won't display in only one row anymore
(Because there aren't spaces between Chinese words usually and the lore books only make a new line when meeting a space, the Chinese localization of the lore book contents won't get any new lines)

更改内容：
为无翻译的字符串添加了翻译
修复了语言文件中的一些错误（错字、错译、马虎等）
优化了剧情书翻译，书里的内容不会再显示在同一行里了

第一次尝试汉化一个模组。如果有中国玩家能看到这段话，那么欢迎对我的汉化文本提出建议。
Current status: Major changes finished; ready to merge. 
@HbmMods

Commit:
fix

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/java/com/hbm/inventory/fluid/Fluids.java
#	src/main/java/com/hbm/inventory/recipes/ChemplantRecipes.java
#	src/main/java/com/hbm/inventory/recipes/CrystallizerRecipes.java
#	src/main/java/com/hbm/inventory/recipes/ElectrolyserFluidRecipes.java
#	src/main/java/com/hbm/tileentity/machine/TileEntityElectrolyser.java
#	src/main/java/com/hbm/tileentity/machine/TileEntityMachineCyclotron.java
#	src/main/java/com/hbm/tileentity/machine/oil/TileEntityMachineLiquefactor.java
#	src/main/java/com/hbm/tileentity/machine/oil/TileEntityMachineSolidifier.java
#	src/main/resources/assets/hbm/lang/zh_CN.lang

Commit:
lang alter

Commit:
lang fix

Commit:
cyclofix

Commit:
fluidfix

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/java/com/hbm/inventory/recipes/CompressorRecipes.java
#	src/main/java/com/hbm/tileentity/machine/TileEntityMachineCompressor.java
#	src/main/resources/assets/hbm/lang/zh_CN.lang

Commit:
大空术士的时间逆转魔术！

Commit:
made overdrive cheaper

Commit:
The Utilitarian's Favorite

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
changelog and small tweaks

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/java/com/hbm/inventory/fluid/Fluids.java

Commit:
final tweaks

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'origin'

Commit:
final fixes

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/resources/assets/hbm/lang/zh_CN.lang

Commit:
changelog, gascent, nuke kits

Commit:
ver, gc mud recipe

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge pull request #10 from OlorinIV/origin

Origin

Commit:
gc&c fixes

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
暴风雨前的宁静

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/java/com/hbm/inventory/recipes/AssemblerRecipes.java

Commit:
outgasser recipe rework

Commit:
补充

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
小更改与防删

Commit:
more fixes and tweaks

Commit:
more fixes and tweaks

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
more pwr coolants

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/java/com/hbm/tileentity/machine/TileEntityMachineGasCent.java
#	src/main/resources/assets/hbm/lang/zh_CN.lang

Commit:
gas cent cleanup

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge pull request #11 from OlorinIV/origin

Origin

Commit:
upgrade fix

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/java/com/hbm/world/feature/BedrockOre.java

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge pull request #12 from OlorinIV/origin

Origin

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'origin'

# Conflicts:
#	gradle.properties
#	src/main/java/com/hbm/blocks/machine/pile/BlockGraphiteFuel.java

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
diode texture fix(?)

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/java/com/hbm/inventory/recipes/ChemplantRecipes.java

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge pull request #13 from OlorinIV/origin

Origin

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge pull request #14 from OlorinIV/origin

Origin

Commit:
pwr post assembly bang fix

Commit:
steel furnace and cyclotron fix

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
crystallizer and gascent recipe tws

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/java/com/hbm/inventory/fluid/Fluids.java

Commit:
sa ore plut-nept&version bump

Commit:
turned out it was a very bad idea

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'origin'

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/java/com/hbm/handler/HazmatRegistry.java
#	src/main/java/com/hbm/items/weapon/ItemCrucible.java
#	src/main/java/com/hbm/main/ModEventHandler.java

Commit:
后门疑似有点太多了

Commit:
rollback shadyutil test

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
chungus eff boost

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/java/com/hbm/inventory/recipes/AssemblerRecipes.java
#	src/main/java/com/hbm/inventory/recipes/ChemplantRecipes.java
#	src/main/java/com/hbm/inventory/recipes/loader/SerializableRecipe.java
#	src/main/java/com/hbm/items/special/ItemStarterKit.java
#	src/main/java/com/hbm/main/CraftingManager.java
#	src/main/java/com/hbm/main/ModEventHandler.java
#	src/main/java/com/hbm/main/ModEventHandlerClient.java
#	src/main/resources/assets/hbm/lang/zh_CN.lang

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
new version tweaks

Commit:
Merge branch 'origin'

# Conflicts:
#	src/main/java/com/hbm/inventory/fluid/Fluids.java
#	src/main/java/com/hbm/inventory/recipes/ChemplantRecipes.java
#	src/main/java/com/hbm/world/feature/BedrockOre.java
#	src/main/resources/assets/hbm/lang/zh_CN.lang

Commit:
Update ChemplantRecipes.java

Commit:
fluid fix

Commit:
arc furnace speedup, version bump

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'refs/heads/origin'

# Conflicts:
#	src/main/java/com/hbm/tileentity/machine/TileEntityMachineArcFurnaceLarge.java

Commit:
changelog

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'refs/heads/origin'

Commit:
typo fix

Commit:
radiolysis final

Commit:
recipes and pouring speed

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'refs/heads/origin'

# Conflicts:
#	src/main/java/com/hbm/inventory/fluid/Fluids.java
#	src/main/java/com/hbm/main/ModEventHandler.java
#	src/main/java/com/hbm/tileentity/machine/TileEntityMachineArcFurnaceLarge.java

Commit:
acid recipe unify

Commit:
version bump and texture fix

Commit:
random qol improvements

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'refs/heads/origin'

# Conflicts:
#	src/main/java/com/hbm/inventory/recipes/ChemplantRecipes.java
#	src/main/java/com/hbm/tileentity/machine/TileEntityMachineCrystallizer.java
#	src/main/resources/assets/hbm/models/machines/acidizer.obj

Commit:
备注

Commit:
焊接台黑片与EtF钍矿适配

Commit:
大范围平衡其一

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'refs/heads/origin'

# Conflicts:
#	src/main/java/com/hbm/blocks/generic/BlockGrate.java
#	src/main/java/com/hbm/tileentity/machine/TileEntityMachineExcavator.java
#	src/main/java/com/hbm/tileentity/machine/TileEntityMachineGasCent.java

Commit:
Merge branch 'HbmMods:master' into origin

Commit:
Merge branch 'refs/heads/origin'

# Conflicts:
#	src/main/java/com/hbm/main/ModEventHandler.java
#	src/main/java/com/hbm/tileentity/machine/TileEntityMachineCyclotron.java

OlorinIV committed:
合并清理

OlorinIV committed:
Merge branch 'HbmMods:master' into origin

OlorinIV committed:
Merge branch 'refs/heads/origin'

Creeper-banner committed:
Rereate gradle.yml

Creeper-banner committed:
Merge pull request #5 from HbmMods/master

yippie

OlorinIV committed:
Merge branch 'HbmMods:master' into origin

OlorinIV committed:
微调、修复

Creeper-banner committed:
Merge branch 'OlorinIV:master' into master

OlorinIV committed:
Merge branch 'HbmMods:master' into origin

OlorinIV committed:
吔屎啦陨石地牢

Creeper-banner committed:
Merge branch 'OlorinIV:master' into master

OlorinIV committed:
u win

OlorinIV committed:
修复

Creeper-banner committed:
Merge branch 'OlorinIV:master' into master

OlorinIV committed:
Merge branch 'HbmMods:master' into origin

OlorinIV committed:
红房间改红石火把

Creeper-banner committed:
Merge branch 'OlorinIV:master' into master

OlorinIV committed:
Merge branch 'HbmMods:master' into origin

OlorinIV committed:
NTMC!

Creeper-banner committed:
Merge branch 'HbmMods:master' into origin

Creeper-banner committed:
Merge branch 'master' into origin

Creeper-banner committed:
Merge branch 'OlorinIV:master' into master

Creeper-banner committed:
Merge pull request #15 from OlorinIV/origin

Origin

Creeper-banner committed:
Merge branch 'master' into master

Creeper-banner committed:
Merge pull request #16 from Creeper-banner/master

Bump

Creeper-banner committed:
Revert "Bump"

Creeper-banner committed:
Merge pull request #17 from OlorinIV/revert-16-master

Revert "Bump"

Creeper-banner committed:
Update ChemplantRecipes.java

Creeper-banner committed:
Create gradle.yml

Creeper-banner committed:
Update gradle.yml

Creeper-banner committed:
Merge pull request #18 from Creeper-banner/master

Update ChemplantRecipes.java

Creeper-banner committed:
Update ElectrolyserMetalRecipes.java

Creeper-banner committed:
Update ElectrolyserFluidRecipes.java

Creeper-banner committed:
Update TileEntityElectrolyser.java

Creeper-banner committed:
Update TileEntityElectrolyser.java

Creeper-banner committed:
Merge pull request #7 from Creeper-banner/exp

Exp

Creeper-banner committed:
Delete .github/workflows/gradle.yml

Creeper-banner committed:
Merge branch 'OlorinIV:master' into master

Creeper-banner committed:
Merge pull request #19 from Creeper-banner/master

补锅#2

Creeper-banner committed:
Merge branch 'HbmMods:master' into master

Creeper-banner committed:
Update changelogsub

Creeper-banner committed:
Merge branch 'HbmMods:master' into master

Creeper-banner committed:
chemplant electrolysis update: heavywater -> 1600 deuterium

Creeper-banner committed:
Update changelogsub

Creeper-banner committed:
Merge branch 'HbmMods:master' into master

Creeper-banner committed:
Changes new bedrock ore recipe duration: 200 -> 160 for ore processing and 100 -> 80 for washing

Creeper-banner committed:
Combination furnace max heat 100,000 -> 125,000

Creeper-banner committed:
Revert

Creeper-banner committed:
Update changelogsub

OlorinIV committed:
Update list

OlorinIV committed:
岂不闻天无绝人之路！

OlorinIV committed:
Update changelogsub

Creeper-banner committed:
Merge branch 'HbmMods:master' into master

Creeper-banner committed:
Merge branch 'HbmMods:master' into master

Creeper-banner committed:
Merge pull request #1 from OlorinIV/master

NTMC migration

Creeper-banner committed:
Create gradle.yml

Creeper-banner committed:
Update gradle.yml

Creeper-banner committed:
Update TileEntityMachineOreSlopper.java

Creeper-banner committed:
Delete .github/workflows/gradle.yml

Creeper-banner committed:
Create gradle.yml

Creeper-banner committed:
Update changelogsub

Creeper-banner committed:
Delete .github/workflows directory

Creeper-banner committed:
Merge branch 'HbmMods:master' into master

Creeper-banner committed:
Merge branch 'HbmMods:master' into ntmc

Creeper-banner committed:
More chemical plant fractioning (and some more)

Creeper-banner committed:
Merge pull request #3 from Creeper-banner/master

More chemical plant fractioning (and some more)

Creeper-banner committed:
Merge pull request #20 from Creeper-banner/ntmc

Changes from my repo

Creeper-banner committed:
Update zh_CN.lang

LSK-LW commented on src/main/resources/assets/hbm/lang/zh_CN.lang:
```diff
@@ -5279,6 +5492,8 @@ tile.pwr_fuel.name=PWR燃料棒
 tile.pwr_fuel.desc=增加压水堆燃料容量$位置：网格，用于增加反应效率
 tile.pwr_heatex.name=PWR换热器
 tile.pwr_heatex.desc=将核心热量转移到外壳$位置：任意
+tile.pwr_heatsink.name=PWR热阱
```
PWR Heatsink

Creeper-banner replied:
这个……被某个人翻译成“热阱”，并且已经在（极小范围内？）使用。
虽然我觉得“热容器”更贴切……

LSK-LW commented on src/main/resources/assets/hbm/lang/zh_CN.lang:
achievement.redRoom.desc=?
achievement.redRoom=The Other Side
analyze.basic1=
analyze.basic2=More documentation can be found on the wiki.
analyze.basic3=
analyze.dummy=Type: Dummy Block
analyze.error=This block's compatibility has not been set properly, this should be reported as a bug!
analyze.name=Name: %s
analyze.noInfo=No additional information.
book_lore.book_iodine.page.0=alright you will not believe this, but old man weathervane finally managed to show up again since he left two weeks ago and what's more surprising is the fact that he actually decided to spill the beans on what they were doing in the canyon:
book_lore.book_flower.page.0=remember when i mentioned in my first memo that the compound is mostly anorganic? well guess what, the old man shared the fourth ingredient: ipomoea nil, a genus of flower. morning glory! it might be due to its low sulfur content, whatever might be the case,
container.furnaceBrick=Bricked Furnace
container.launchPadRusted=Launch Pad
container.rttyLogic=Redstone-over-Radio Logic Receiver
desc.gui.rttyLogic.cond0=Signal §6LESS THAN§r Constant
desc.gui.rttyLogic.cond1=Signal §6LESS THAN OR EQUAL TO§r Constant
desc.gui.rttyLogic.cond2=Signal §6GREATER THAN OR EQUAL TO§r Constant
desc.gui.rttyLogic.cond3=Signal §6GREATER THAN§r Constant
desc.gui.rttyLogic.cond4=Signal §6EQUAL TO§r Constant
desc.gui.rttyLogic.cond5=Signal §6NOT EQUAL TO§r Constant
desc.gui.rttyLogic.cond6=Signal §6MATCHES§r String
desc.gui.rttyLogic.cond7=Signal §6DOES NOT MATCH§r String
desc.gui.rttyLogic.cond8=Signal §6CONTAINS§r String
desc.gui.rttyLogic.cond9=Signal §6DOES NOT CONTAIN§r String
entity.entity_ntm_fbi_drone.name=FBI Drone
fluid.rad_lava_fluid=Volcanic Lava
hadron.hopper2=§eSingle Mode:$Only allows one item per slot.
item.circuit_star_component.card.name=StarControl Extension Card
item.circuit_star_piece.board_converter.name=SC - Voltage Converter
item.circuit_star_piece.card_board.name=SC - Extension Card Board
item.circuit_star_piece.card_processor.name=SC - Extension Card Processor
item.drone.patrol.desc=Follows a route of waypoints$Tries to load/unload every time it reaches a drone crate$Right click on top of a drone crate to place.
item.drone.patrol_chunkloading.desc=Follows a route of waypoints$Tries to load/unload every time it reaches a drone crate$Right click on top of a drone crate to place.$Loads chunks!
item.drone.patrol_express.desc=Follows a route of waypoints$Tries to load/unload every time it reaches a drone crate$Right click on top of a drone crate to place.
item.drone.patrol_express_chunkloading.desc=Follows a route of waypoints$Tries to load/unload every time it reaches a drone crate$Right click on top of a drone crate to place.$Loads chunks!
item.drone.request.desc=Used by logistics drone docks to spawn logistics drones$Drones will follow logistics drone waypoints$After completing a request, the drone will return to the dock.
item.key_red.key_red_cracked.P11=§4???§r
item.ore.hematite=Hematite
item.ore.malachite=Malachite
item.ore.neodymium=Neodymium
item.ore.thorium232=Thorium
item.page_of_.page1.name=Page 1
item.page_of_.page2.name=Page 2
item.page_of_.page3.name=Page 3
item.page_of_.page4.name=Page 4
item.page_of_.page5.name=Page 5
item.page_of_.page6.name=Page 6
item.page_of_.page7.name=Page 7
item.page_of_.page8.name=Page 8
item.stamp_book.printing1.name=Printing Press Stamp (Part 1)
item.stamp_book.printing2.name=Printing Press Stamp (Part 2)
item.stamp_book.printing3.name=Printing Press Stamp (Part 3)
item.stamp_book.printing4.name=Printing Press Stamp (Part 4)
item.stamp_book.printing5.name=Printing Press Stamp (Part 5)
item.stamp_book.printing6.name=Printing Press Stamp (Part 6)
item.stamp_book.printing7.name=Printing Press Stamp (Part 7)
item.stamp_book.printing8.name=Printing Press Stamp (Part 8)
radar.target.abm=Anti-Ballsitic Missile
radar.target.custom10=Size 10 Custom Missile
radar.target.custom1015=Size 10/15 Custom Missile
radar.target.custom15=Size 15 Custom Missile
radar.target.custom1520=Size 15/20 Custom Missile
radar.target.custom20=Size 20 Custom Missile
radar.target.doomsday=Doomsday Missile
radar.target.shuttle=Reliant Robin Space Shuttle
radar.target.tier0=Tier 0 Missile
radar.target.tier1=Tier 1 Missile
radar.target.tier2=Tier 2 Missile
radar.target.tier3=Tier 3 Missile
radar.target.tier4=Tier 4 Missile
shape.ntmpipe=Pipe
shape.shell=Shell
shape.wireFine=Wires
shape.wireDense=Dense Wire
shape.wiresDense=Dense Wires
tile.machine_funnel.desc=Automatically compresses items in a 2x2 or 3x3 crafting grid$Top: Input$Bottom: Output$Side: Output for clearing ingredients
tile.oil_spill.name=Oil Spill
tile.rbmk_loader.desc=Allows RBMKs to have both water and steam connections at the bottom$Place one water pipe below the RBMK column, then the connector,$then connect the steam duct to the connector.
tile.rbmk_steam_inlet.desc=Inserts water into RBMK columns if ReaSim boilers are enabled$Connects to RBMK columns sideways
tile.rbmk_steam_outlet.desc=Extracts super dense steam from RBMK columns if ReaSim boilers are enabled$Connects to RBMK columns sideways
upgrade.acid=Acid required %s
upgrade.burn=Burn %smb/t for %sHE
upgrade.consumption=Consumption %s
upgrade.coolantConsumption=Coolant Consumption %s
upgrade.delay=Process time %s
upgrade.efficiency=Efficiency %s
upgrade.fortune=Fortune %s
upgrade.overheatChance=Overheat chance %s
upgrade.range=Range %s
upgrade.gui.title=§lAcceptable Upgrades:§r
upgrade.gui.afterburner= * §dAfterburner§r: Stacks to level %s
upgrade.gui.effectiveness= * §aEffectiveness§r: Stacks to level %s
upgrade.gui.overdrive= * §7Overdrive§r: Stacks to level %s
upgrade.gui.power= * §9Power-Saving§r: Stacks to level %s
upgrade.gui.speed= * §4Speed§r: Stacks to level %s

Creeper-banner replied:
这些的话会在一段时间之后补全。但是这里面有些东西好像是我已经翻过的啊？

LSK-LW replied:
恩..这是idea插件对比英语文件缺少的部分

LSK-LW commented on src/main/resources/assets/hbm/lang/zh_CN.lang:
```diff
@@ -313,60 +313,60 @@ book.starter.page18=v?r只是一个被困在"传声头像"的§o《一生一次
-book_lore.test.page.3=3
-book_lore.test.page.4=4
+book_lore.test.page.0=>你好，在我开始之前，让我澄 清我不是同性恋。 >做我 >今天晚上 >18 >和我最好的朋友和我的父母出去玩了几天 >我们已经是最好的朋友一年了 >喝了点酒，玩了很多电子游戏，还点了一份披萨 >我们玩得很开心 >在晚上的某个时候，比如9点左右，他开了一个非常有趣的玩笑，我不记得了，但我知道这让我们俩都笑得很厉害 >没有我想我用右手摸了摸他那半卷曲的黑发，然后叫他一个有趣的男孩 >他脸红了 >我意识到我很慌张 >我们有点亲密 >突然他吻了我，出于某种原因我又吻了他 >我们做爱 >拥抱在一起睡着了 >半夜醒来时，他的头贴在我的胸部和颈部 >感觉很好，但我不是同性恋
+book_lore.test.page.1=他在我怀里睡着时，我正在打 字。我怎么才能让我最好的 朋友失望呢？ 我不想成为[编辑]/b/
```
请求重翻 这两段。。

Creeper-banner replied:
> 请求重翻 这两段。。

>hello anons before i begin let me clarify that i'm not gay. 
>哈喽 在我开始之前 我得先说一句 我不是同性恋

>Be me 
>做我

>This night 
>这个晚上

>18 
>18

>At hanging out with my best friend with my parents gone for a few days 
>爸妈不在 我和铁哥们去玩了几天

>We've been best friends for a year now 
>我们成了铁哥们已经有一年了

>Been drinking a bit and playing lots of video games and ordered a pizza 
>喝了点酒 打了一堆游戏 还点了个披萨

>We were having a blast 
>我们玩得正高兴

>At a certain point in the night like around 9:00 he makes a really funny joke that I don't remember but I 

know that it made us both laugh really hard 
>晚上 在一个什么时候 大概九点吧 他讲了个很好笑的笑话 那笑话我忘了 但是我记着 那笑话让我俩笑疯了

> With out thinking I brush my right hand through his semi-curly black hair and call him a funny boy 
>没来得及细想 我拿手捋了捋他的头发 黑色的微卷的头发 还说他真有意思

>He blushes
>他脸红了 

>I realize I'm feeling flustered
>我意识到自己慌了

>We're kinda close 
>我们有点太亲密了

>All of the sudden he kisses me and for some reason I kiss him back
>突然他亲了我 不知道为什么 我亲回去了 

>We make love 
>我们做爱了

>Cuddle together and fall asleep 
>抱在一起睡着了

>Wake up in the middle of the night with his head snuggled up on my chest and neck area 
>半夜我醒了 发现他脑袋贴着我的胸和脖子

>It feels nice but I'm not a homosexual
>这种感觉很奇妙 但我真的不是同性恋

I'm typing this as he's asleep in my arms. How do I let my best friend down nicely? I don't want to be a [redacted] /b/
我打这些字的时候他正在我怀里睡觉。我怎么和我哥们交代啊？我不想变成[数据删除]

MellowArpeggiation replied:
(⊙ ͜ʖ⊙)

DrIshirozaki replied:
Can't believe they turned github into 4chan

Creeper-banner replied:
> (⊙ ͜ʖ⊙)

Those sentenses are actually written by Bob. I just translated it into Chinese.
(but... quite exciting don't they)

Creeper-banner committed:
Update zh_CN.lang

Creeper-banner committed:
Carelessness fix

Creeper-banner commented:
Update:
added translations for remaining untranslated strings
Fixed more mistranslations

更新：
补全了部分翻译
修复了更多错译

TODO: translations for all lore books
           翻译所有的剧情书

同样地，欢迎能看到这段话的中国玩家对我的翻译提出建议。

Creeper-banner committed:
Update zh_CN.lang

Creeper-banner commented:
Added translations for all lore books.

Creeper-banner committed:
Merge branch 'HbmMods:master' into master

Creeper-banner committed:
Update list

Creeper-banner committed:
Merge branch 'HbmMods:master' into ntmc

Creeper-banner committed:
Merge pull request #2 from OlorinIV/master

Merge original NTMC

Creeper-banner committed:
Converters won't eat up energy anymore!

Creeper-banner committed:
Merge pull request #4 from Creeper-banner/master

Converters won't eat up energy anymore!

Creeper-banner committed:
Night vision MK2 (self-charging)

Creeper-banner committed:
Merge branch 'ntmc' of https://github.com/Creeper-banner/NTMC.git into ntmc

Creeper-banner committed:
Merge pull request #21 from Creeper-banner/ntmc

Merge CrpBnrz NTMC

LSK-LW reviewed (approved):
Good!

Creeper-banner committed:
Merge branch 'HbmMods:master' into bob

Kal19801 commented:
挺不错

Creeper-banner committed:
Update zh_CN.lang

Creeper-banner committed:
Super hot steam as ICF coolant
(and switched to BMCLAPI mirror)

Creeper-banner committed:
Merge pull request #22 from Creeper-banner/ntmc

Super hot steam as ICF coolant

Creeper-banner committed:
Update zh_CN.lang

NarekoMichigami810 commented:
lang文件中在中文括号内的文本会在游戏中消失，表现为下图，请尽量使用英文括号带上空格，辛苦了
![中文括号](https://github.com/user-attachments/assets/61f48035-03cc-4a17-b67d-40186e1c10e1)

Creeper-banner commented:
> lang文件中在中文括号内的文本会在游戏中消失，表现为下图，请尽量使用英文括号带上空格，辛苦了 ![中文括号](https://private-user-images.githubusercontent.com/169655057/352709075-61f48035-03cc-4a17-b67d-40186e1c10e1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIwNjc3MzcsIm5iZiI6MTcyMjA2NzQzNywicGF0aCI6Ii8xNjk2NTUwNTcvMzUyNzA5MDc1LTYxZjQ4MDM1LTAzY2MtNGExNy1iNjdkLTQwMTg2ZTFjMTBlMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyN1QwODAzNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMTA4ZDIzODY1OGY0ZDI0ODEyODNiNGI4MDJkYmFiMGQwZDBjNzJkYjllZDI2MWUyMjEyMjBiNjJiYTEyMmMzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XARkkuYE9xoAg0yZACZkj7ALZxMnakbw7AvK2fQMi3g)

安装Optifine可以解决这个问题。括号的话……再说。

TheGreatAndPowerfulArclight commented:
“粉色云”和“粉色的云”都应译为“粉雾”，不过之前我已经在HBM的Discord服务器里反馈过了；Spark系列电池可以译为“闪闪魔法电池”，辐马里就是这么翻译的，例如Spark反物理储能装置可以译为”闪闪反物理储能装置“，Spark电池组译为”闪闪魔法电池组“（或许？）
![Uploading 屏幕截图 2024-07-27 164604.png…]()

TheGreatAndPowerfulArclight commented:
<img width="781" alt="屏幕截图 2024-07-27 164604" src="https://github.com/user-attachments/assets/a833410e-f8a9-4056-bff9-9bdb4eb7f724">

TheGreatAndPowerfulArclight commented:
“最后战线—呃，算了吧”该成就中的Final Frontier结合语境应译为“最后边疆”，“the Final Frontier"即指宇宙。例如ST里的这句话：“Space, the final frontier.”（宇宙，最后的边疆。）

TheGreatAndPowerfulArclight commented:
还有，BF炸弹译为野火炸弹或许更好，Balefire在别处都译为野火，只在炸弹这里缩写成BF有点怪

Creeper-banner commented:
> 还有，BF炸弹译为野火炸弹或许更好，Balefire在别处都译为野火，只在炸弹这里缩写成BF有点怪

这条和上面那条倒是可以考虑。Spark电池已经算是流传比较广的翻译了，再改了反倒不合适。
另外其实很多成就的名称和描述的翻译都不算恰当，但是似乎涉及一些外网的梗和典故，不太懂。

TheGreatAndPowerfulArclight commented:
> > 还有，BF炸弹译为野火炸弹或许更好，Balefire在别处都译为野火，只在炸弹这里缩写成BF有点怪
> 
> 这条和上面那条倒是可以考虑。Spark电池已经算是流传比较广的翻译了，再改了反倒不合适。 另外其实很多成就的名称和描述的翻译都不算恰当，但是似乎涉及一些外网的梗和典故，不太懂。

OK

TheGreatAndPowerfulArclight commented:
> > 还有，BF炸弹译为野火炸弹或许更好，Balefire在别处都译为野火，只在炸弹这里缩写成BF有点怪
> 
> 这条和上面那条倒是可以考虑。Spark电池已经算是流传比较广的翻译了，再改了反倒不合适。 另外其实很多成就的名称和描述的翻译都不算恰当，但是似乎涉及一些外网的梗和典故，不太懂。

不过粉雾的翻译得改下，这个的确是错译了

Creeper-banner committed:
Update zh_CN.lang: achievement, minor changes

Creeper-banner committed:
Update zh_CN.lang

Creeper-banner commented:
> > > 还有，BF炸弹译为野火炸弹或许更好，Balefire在别处都译为野火，只在炸弹这里缩写成BF有点怪
> > 
> > 
> > 这条和上面那条倒是可以考虑。Spark电池已经算是流传比较广的翻译了，再改了反倒不合适。 另外其实很多成就的名称和描述的翻译都不算恰当，但是似乎涉及一些外网的梗和典故，不太懂。
> 
> 不过粉雾的翻译得改下，这个的确是错译了

粉雾和野火炸弹改了，部分成就的名称和描述也改了。应该没有需要大改的地方了。
但是最近Bob没活动。

Creeper-banner committed:
Update zh_CN.lang

TheGreatAndPowerfulArclight commented:
> > > > 还有，BF炸弹译为野火炸弹或许更好，Balefire在别处都译为野火，只在炸弹这里缩写成BF有点怪
> > > 
> > > 
> > > 这条和上面那条倒是可以考虑。Spark电池已经算是流传比较广的翻译了，再改了反倒不合适。 另外其实很多成就的名称和描述的翻译都不算恰当，但是似乎涉及一些外网的梗和典故，不太懂。
> > 
> > 
> > 不过粉雾的翻译得改下，这个的确是错译了
> 
> 粉雾和野火炸弹改了，部分成就的名称和描述也改了。应该没有需要大改的地方了。 但是最近Bob没活动。

辛苦辛苦

TheGreatAndPowerfulArclight commented:
"它永远不能防止："
这句描述有点怪，虽然能理解其含义，但读起来似乎不是很通顺，并不符合中文用语习惯，译为“它对以下危害无效：”如何？

Creeper-banner committed:
Update zh_CN.lang

Creeper-banner commented:
> "它永远不能防止：" 这句描述有点怪，虽然能理解其含义，但读起来似乎不是很通顺，并不符合中文用语习惯，译为“它对以下危害无效：”如何？

改成了“无法防护”，应该也说得通。

Creeper-banner committed:
Update list

Creeper-banner committed:
Merge pull request #6 from OlorinIV/master

Merge original NTMC

Creeper-banner committed:
Create gradle.yml

Creeper-banner committed:
test

Creeper-banner committed:
Merge branch 'master' of https://github.com/Creeper-banner/NTMC.git

Creeper-banner committed:
fluid test

Creeper-banner committed:
Merge pull request #7 from Creeper-banner/master

Master

Creeper-banner committed:
Update gradle.yml

Creeper-banner committed:
Na/Sr/Nd ingot/block preparations

Creeper-banner committed:
Update ModBlocks.java

Creeper-banner committed:
My Translations

Creeper-banner committed:
Merge branch 'ntmc' of https://github.com/Creeper-banner/NTMC.git into ntmc

Creeper-banner committed:
Merge branch 'bob' of https://github.com/Creeper-banner/NTMC.git into ntmc

# Conflicts:
#	src/main/resources/assets/hbm/lang/zh_CN.lang

Creeper-banner committed:
Update gradle.yml

Creeper-banner committed:
Textures for modded cryogel and Na/Sr/Nd ingot/block

Creeper-banner committed:
Today's last major part of update

Creeper-banner committed:
Carelessness

Creeper-banner committed:
Carelessness #2

Creeper-banner committed:
At least I remembered to import packages since then

Creeper-banner committed:
Final works

Creeper-banner committed:
Bump version

Creeper-banner committed:
Complement

Creeper-banner committed:
Merge pull request #23 from Creeper-banner/ntmc

Merge changes from my repo

TheGreatAndPowerfulArclight commented:
> > "它永远不能防止：" 这句描述有点怪，虽然能理解其含义，但读起来似乎不是很通顺，并不符合中文用语习惯，译为“它对以下危害无效：”如何？
> 
> 改成了“无法防护”，应该也说得通。

没毛病

Creeper-banner committed:
Merge branch 'HbmMods:master' into master

Creeper-banner committed:
Update list

Creeper-banner committed:
Merge branch 'HbmMods:master' into ntmc

Creeper-banner committed:
Merge branch 'HbmMods:master' into master

Creeper-banner committed:
Merge branch 'HbmMods:master' into ntmc

Creeper-banner committed:
Merge pull request #9 from OlorinIV/master

O4

Creeper-banner committed:
Small tweaks

Creeper-banner committed:
Combination recipe for bedrock ore tweaks

Creeper-banner committed:
Revert for Actions

Creeper-banner committed:
My bad

Creeper-banner committed:
Combination recipes test

Creeper-banner committed:
ouch

Creeper-banner committed:
Small complement

Creeper-banner committed:
Dict stack fixes

Creeper-banner committed:
I would close my IME next time

Creeper-banner committed:
ouch

Creeper-banner committed:
CMB drillbits

Creeper-banner committed:
textures

Creeper-banner committed:
Merge branch 'HbmMods:master' into ntmc

Creeper-banner committed:
Maskman spawn

Creeper-banner committed:
Merge branch 'ntmc' of https://github.com/Creeper-banner/NTMC.git into ntmc

Creeper-banner committed:
Mask man spawn texture & localization

Creeper-banner committed:
changelogsub & list

Creeper-banner committed:
Alternative fullerene production

Creeper-banner committed:
ouch

Creeper-banner committed:
ouch 2

Creeper-banner committed:
ouch 3

Creeper-banner committed:
I felt myself got fooled

Creeper-banner committed:
ouch 4

Creeper-banner committed:
textures, tweaks about recipes involving CFT

Creeper-banner committed:
Version bump

Creeper-banner committed:
Merge branch 'HbmMods:master' into bob

Creeper-banner committed:
Merge branch 'bob' of https://github.com/Creeper-banner/NTMC.git into ntmc

# Conflicts:
#	src/main/java/com/hbm/tileentity/machine/oil/TileEntityMachineLiquefactor.java
#	src/main/java/com/hbm/tileentity/machine/oil/TileEntityMachineSolidifier.java

Creeper-banner committed:
Update zh_CN.lang

Creeper-banner committed:
Update zh_CN.lang

Creeper-banner committed:
Update zh_CN.lang

Creeper-banner committed:
plushie

Creeper-banner committed:
Merge branch 'ntmc' into bob

Creeper-banner commented:
https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/pull/1615
